providers/github/connection/platform.go

```diff
@@ -58,20 +61,40 @@ type RepositoryId struct {
 func (c *GithubConnection) PlatformInfo() (*inventory.Platform, error) {
 	conf := c.asset.Connections[0]
 	if orgId := conf.Options["organization"]; orgId != "" {
-		return GithubOrgPlatform, nil
+		return NewGithubOrgPlatform(orgId), nil
 	}
 
 	if userId := conf.Options["user"]; userId != "" {
-		return GithubUserPlatform, nil
+
+		return NewGithubUserPlatform(userId), nil
 	}
 
 	if repo := conf.Options["repository"]; repo != "" {
-		return GithubRepoPlatform, nil
+		owner := conf.Options["owner"]
```

Comment:
Do we check somewhere else, that "owner" is set in the options?

Reply:
That is a flexible entry the config just sets the schema

Reply:
we may need more checking, but that would need to be done earlier https://github.com/mondoohq/cnquery/blob/main/providers/github/provider/provider.go#L167-L176